404.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+---
+
+<div class="row">
+    <div class="container my-5 py-5">
+        <div class="row">
+            <div class="offset-lg-2 col-lg-7 mt-md-4">
+                <img alt="" width="40" height="40" src="assets/alert.svg">
+                <h1 class="h2 mb-3 pt-4 mt-3">Something’s missing!</h1>
+            </div>
+            <div class="offset-lg-2 col-lg-6 mb-md-4">
+                <p class="mb-3 pb-3 text-info">The page you’re looking for doesn’t exist or was removed. We suggest you
+                    go back home.</p>
+                <div class="d-flex flex-column flex-lg-row mb-5 pb-5">
```

Comment:
After https://github.com/compilerla/compiler.la/pull/134 and https://github.com/compilerla/compiler.la/pull/135 are merged, I believe the `mb-5` and `pb-5` can also be removed.